src/elements/content-sidebar/__tests__/SidebarPanels.test.js

```diff
@@ -11,18 +11,23 @@ jest.mock('../SidebarUtils');
 describe('elements/content-sidebar/SidebarPanels', () => {
     const getWrapper = ({ path = '/', ...rest } = {}) =>
         mount(
-            <MemoryRouter initialEntries={[path]} keyLength={0}>
-                <SidebarPanels
-                    file={{ id: '1234' }}
-                    hasActivity
-                    hasDetails
-                    hasMetadata
-                    hasSkills
-                    hasVersions
-                    isOpen
-                    {...rest}
-                />
-            </MemoryRouter>,
+            <SidebarPanels
+                file={{ id: '1234' }}
+                hasActivity
+                hasDetails
+                hasMetadata
+                hasSkills
+                hasVersions
+                isOpen
+                {...rest}
+            />,
+            {
```

Comment:
Curious, what was the reason for this change?

Reply:
`setProps()` can only be called on the root, which was `MemoryRouter` in the previous setup. So, this is more of a workaround to call `setProps()` on `SidebarPanels` and thus trigger `componentDidUpdate()`.